### PR TITLE
refactor: change common expect error messages to constants

### PR DESCRIPTION
**Pull request by asmie**

### CODE REVIEW

File: src/tests/conformance/post_handshake/cmd/transaction.rs [hunk 1]
```diff
@@ -86,5 +85,5 @@ async fn c012_TXN_submit_txn_and_expect_to_receive_it() {
     synthetic_node_rx.shut_down().await;
     synthetic_node_tx.shut_down().await;
     kmd.stop().expect("unable to stop the kmd instance");
```
Comment (Rqnsom) [inline src/tests/conformance/post_handshake/cmd/transaction.rs:87]: We can update `kmd` also with (these already exist in the const.rs):
```
/// Error message when building a kmd instance fails.
pub const ERR_KMD_BUILD: &str = "unable to build the kmd instance";

/// Error message when a kmd instance fails to stop.
pub const ERR_KMD_STOP: &str = "unable to stop the kmd instance";
```

File: src/tests/conformance/post_handshake/broadcast/proposalpayload.rs [hunk 2]
```diff
@@ -11,25 +18,23 @@ async fn c007_PROPOSAL_PAYLOAD_expect_after_connect() {
...
 
-    let net_addr = node.net_addr().expect("network address not found");
+    let net_addr = node.net_addr().expect(ERR_NODE_ADDR);
 
     // Connect to the node and initiate the handshake.
     synthetic_node
         .connect(net_addr)
         .await
-        .expect("unable to connect");
+        .expect(ERR_NODE_CONNECT);
```
Comment (Rqnsom) [inline src/tests/conformance/post_handshake/broadcast/proposalpayload.rs:37]: I just noticed:
```suggestion
        .expect(ERR_SYNTH_CONNECT);
```
If we follow the rule: `ERR_<WHO>_<FUNCTION_NAME>`
`Who` is here our synthetic_node, not the node.

**Review by Rqnsom — APPROVED** [review]
LGTM, a few comments

File: src/tests/conformance/post_handshake/cmd/transaction.rs (hunk 1 above)
Comment (asmie) [reply src/tests/conformance/post_handshake/cmd/transaction.rs]: Done.

File: src/tests/conformance/post_handshake/broadcast/proposalpayload.rs (hunk 2 above)
Comment (asmie) [reply src/tests/conformance/post_handshake/broadcast/proposalpayload.rs]: Done.

File: src/tools/constants.rs [hunk 3]
```diff
@@ -38,5 +38,11 @@ pub const ERR_SYNTH_UNICAST: &str = "unable to send a message";
 /// Error message when synthetic node creation fails.
 pub const ERR_SYNTH_BUILD: &str = "unable to build a synthetic node";
 
+/// Error message for a failed connection.
+pub const ERR_SYNTH_CONNECT: &str = "unable to connect to the node";
```
Comment (Rqnsom) [inline src/tools/constants.rs:42]: I think we can delete this one:
`pub const ERR_NODE_CONNECT: &str = "unable to connect to the node";`

**Review by Rqnsom — APPROVED** [review]

File: src/tools/constants.rs (hunk 3 above)
Comment (asmie) [reply src/tools/constants.rs:39]: Deleted for now.

**Pull request merged**